src/collective/contact/facetednav/browser/static/javascript.js

```diff
@@ -25,7 +27,9 @@ contactfacetednav.init = function() {
                     contact.render();
                 });
             }
+            window.setTimeout(function() {
             contactfacetednav.contacts.render();
+            }, 500);
```

Comment:
Why 500? Did you try 0 to see if the delayed call was enough?

Reply:
fixed !